Cargo.toml

```diff
@@ -79,13 +78,13 @@ hardware-signer = ["hwi"]
 async-interface = ["async-trait"]
 electrum = ["electrum-client"]
 # MUST ALSO USE `--no-default-features`.
-use-esplora-reqwest = ["esplora", "reqwest", "reqwest/socks", "futures"]
-use-esplora-ureq = ["esplora", "ureq", "ureq/socks"]
+use-esplora-reqwest = ["esplora", "esplora-client/async", "futures"]
+use-esplora-ureq = ["esplora", "esplora-client/blocking"]
```

Comment:
I think keeping these names is a bit strange given the `ureq`/`reqwest` dependencies have been removed. Maybe these features could also be renamed `use-esplora-async`/`use-esplora-blocking`, while initially keeping the old features around as aliases to not break compatibility?

Reply:
I also like this idea. Add the new feature names, but keep the old ones around for a release with comment in `Cargo.toml` and we'll put in tag changelog that old feature names are deprecated.

Reply:
Apparently there's no way to deprecate a feature, should we just add a note to the changelog?

Reply:
What you did with making the old ureq/reqwest features aliases for the new names is all I was expecting, plus adding your change log note.

Reply:
In line with @tnull point, I was wondering if it doesn't also make sense to rename the `ureq.rs` and `reqwest.rs` file under the `esplora` module to `blocking.rs` and `async.rs` respectively.

Reply:
I did consider it but since the modules are not public it doesn't make any difference for the public API, so it's just an internal thing

Reply:
Hm, but doesn't that mean that there is even less of a reason not to clean it up?

Reply:
Yeah, maybe, I personally don't mind but I'm not against cleaning it up either.

I replied like that because for some reason I though this PR had already been merged by Steve and so I thought it was too late to fix it and I didn't want to open another PR just for that. But since this is still open yeah I can do it!